src/libs/state-history.js

```diff
@@ -0,0 +1,8 @@
+import _ from 'lodash'
+
+
+export const set = newState => { window.history.replaceState(newState, '') }
```

Comment:
Is it worth catching the "over-sized" error per https://app.zenhub.com/workspace/o/databiosphere/saturn-ui/issues/65?

Reply:
It'll throw an exception; for the time being we can just be vigilant, I'm not sure what else we'd do.